migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -50,7 +50,6 @@ CREATE TABLE IF NOT EXISTS transactions (
     signature BLOB NOT NULL,
     validated BOOLEAN NOT NULL,
     bundle_id CHAR(43),
-    sent_to_leader BOOLEAN NOT NULL,
```

Comment:
Since we don't have launched yet, ok to put in the initial migration

Reply:
That was my thinking too.